apps/challenges/migrations/0034_auto_20180330_1024.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please make sure that you name the migration while running the command `makemigrations`